### PR TITLE
Add/post documentation for apiFetch

### PR DESCRIPTION
**Pull request by momotofu**
## Description
Added a POST example in the apiFetch documentation. Addresses issue #19649 

## Types of changes
Documentation

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: packages/blocks/src/api/utils.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { default as tinycolor, mostReadable } from 'tinycolor2';
  */
 import { Component, isValidElement } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
-import { stripHTML } from '@wordpress/dom';
+import { __unstableStripHTML as stripHTML } from '@wordpress/dom';
```
Comment (youknowriad) [inline packages/blocks/src/api/utils.js]: It seems like there's some unrelated changes in this PR?

Comment (momotofu) [reply packages/blocks/src/api/utils.js]: Weird. I'm not sure how these files made it into this PR.

File: packages/api-fetch/README.md [hunk 2]
```diff
@@ -17,9 +17,22 @@ _This package assumes that your code will run in an **ES2015+** environment. If
...
 } );
+
+// POST
+apiFetch( {
+	path: '/wp/v2/posts/1',
+	method: 'POST',
+	headers: {
+		'Content-Type': 'application/json',
+	},
+	body: JSON.stringify( { title: 'New Post Title' } ),
```
Comment (youknowriad) [inline packages/api-fetch/README.md]: I'm pretty sure you can drop the headers and the body here and replace with just:

```suggestion
	data:  { title: 'New Post Title' },
```

Comment (momotofu) [reply packages/api-fetch/README.md]: Yes, good point. I was able verified that. Let's go with the terser solution.

**Review by youknowriad — APPROVED** [review]

**Comment by youknowriad** [comment]
Thanks for the addition
